### PR TITLE
Changes for the release 

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -34,9 +34,9 @@ environment:
     # Note requiring a specific version of stack using STACKVER may fail due to
     # github API limit while checking and upgrading/downgrading to the specific
     # version.
-    #STACKVER: "1.6.5"
+    # STACKVER: "1.6.5"
     STACK_UPGRADE: "y"
-    RESOLVER: "lts-18.18"
+    # RESOLVER: "lts-18.27"
```
Comment (adithyaov) [inline appveyor.yml:39]: Keeping things DRY, resolver will be picked from stack.yaml
We can add another stack.yaml for an SDIST build.

File: dev/MAINTAINING.md [hunk 2]
```diff
@@ -117,11 +117,10 @@
     * Bump the release version in `docs/CONTRIBUTORS.md`.
     * Make sure any third party code included in the release has been listed in
       `docs/Credits.md` and the license is added to the repo.
-    * Bump the release version in `docs/Credits.md`.
```
Comment (adithyaov) [inline dev/MAINTAINING.md:120]: There is no release version in this file.

File: docs/Changelog.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 <!-- See rendered changelog at https://streamly.composewell.com -->
 
+## 0.8.2 (Feb 2022)
+
```
Comment (harendra-kumar) [inline docs/Changelog.md:6]: We can mention the performance fixes for GHC 9.2. The performance will be on par with previous compilers once GHC fixes land in GHC 9.2.2.

File: docs/Changelog.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
+### Notable Internal Changes
```
Comment (harendra-kumar) [inline docs/Changelog.md:7]: We can skip all of these.

**Review by harendra-kumar — COMMENTED** [review]
We can update the build guide with the expectation that GHC 9.2.2 will have the performance fixes.

File: docs/Compiling.md [hunk 5]
```diff
@@ -8,13 +8,19 @@ GHC 8.6 and above are recommended.  For best performance use GHC 8.8 or
...
+GHC 9.0 and GHC 9.2.1 have some performance issues, please see [this
 issue](https://github.com/composewell/streamly/issues/1061) for details.
-However, upcoming releases may fix some of these issues.
+
+However, the fixes required in streamly for the majority of performance
+regressions have been implemented. Please see [this
+issue](https://github.com/composewell/streamly/pull/1462) for details.
+
+The performance will be on par with previous compilers once GHC fixes land in
+GHC 9.2.2.
```
Comment (harendra-kumar) [inline docs/Compiling.md:19]: We can keep it terse.

"GHC 9.0 and GHC 9.2.1 have some performance issues, please see [this
issue](https://github.com/composewell/streamly/issues/1061) for details. 
GHC 9.2.2 is expected to have the fixes that will bring the performance on par with previous versions."

**Review by harendra-kumar — APPROVED** [review]

**Pull request merged**